src/storage/v2/property_store.hpp

```diff
@@ -19,6 +20,45 @@
 
 namespace memgraph::storage {
 
+struct SearchResult {
+  std::uint64_t id;
+  nlohmann::json document;
+  double score;
+};
+
+class ExternalStoreMock {
+ private:
+  std::string INDEX = "";  // placeholder (Tantivy supports multiple indices)
+  std::map<std::string, std::map<std::uint64_t, nlohmann::json>> storage{};  // index_name: {document_id: document}
+
+ public:
+  enum class Consistency : uint8_t { DEFAULT };
+
+  void CreateAllPropsIndex(std::string index_name, LabelId label, std::string tokenizer = "DUMMY_TOKENIZER",
+                           Consistency consistency = Consistency::DEFAULT);
+
+  void CreateIndex(std::string index_name, LabelId label, std::vector<PropertyId> properties,
+                   std::string tokenizer = "DUMMY_TOKENIZER", Consistency consistency = Consistency::DEFAULT);
```

Comment:
What was the point with `Consistency` here (I don't recall exactly)?

Reply:
We’ve got the consistency parameter since it was mentioned in this issue: https://github.com/memgraph/memgraph/issues/1261. We considered it as a Tantivy parameter, but I haven’t found anything like it in the docs. Given that, can we remove it from the function?

Reply:
Resolving (consistency is not in the current mock, nor in mgcxx)